src/styles/overlay-panel.less

```diff
@@ -2,30 +2,50 @@
   display: block;
 }
 
+.catalogs-overlay-panel-wrapper {
+  position: fixed;
```

Comment:
Nit:  Declarations should be alphabetized by property.  And rules should be alphabetized by selector.